sci-physics/rivet/rivet-2.6.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools bash-completion-r1 python-single-r1
+
+DESCRIPTION="Toolkit for validation of Monte Carlo HEP event generators"
+HOMEPAGE="http://rivet.hepforge.org/"
+
+SRC_URI="http://www.hepforge.org/archive/${PN}/${P^}.tar.bz2
+	doc? ( https://rivet.hepforge.org/trac/export/8a05acecd26c18f368f4b748da2d5aa2db46be6c/doc/refs.bib )"
+LICENSE="GPL-2"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc python static-libs"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	dev-libs/boost:=
+	sci-libs/gsl:=
+	sci-physics/fastjet[plugins]
+	sci-physics/hepmc
+	>=sci-physics/yoda-1.5.0[python]
+	python? ( ${PYTHON_DEPS} )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen[latex,dot] )
+	python? ( dev-python/cython[${PYTHON_USEDEP}] )"
+
+S="${WORKDIR}/${P^}"
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_unpack() {
+	unpack "${P^}.tar.bz2"
+
+	if use doc; then
+		# refs.bib is missing in tarball (reported upstream)
+		cp "${DISTDIR}"/refs.bib "${S}"/doc || die
+	fi
+}
+
+src_prepare() {
+	default
+
+	# Install rivet-manual.pdf to docdir intead of pkgdatadir
+	sed -i '/pkgdata_DATA = $(DOCS)/s/pkgdata/doc/' doc/Makefile.am || die
+	eautoreconf
+}
+
+src_configure() {
+	append-cxxflags "-std=c++11"
```

Comment:
Hi Federico, I am testing locally, and it seems that the package itself takes care of adding this flag to the compilation options. What compiler did you use that needed the flag here?

Reply:
I tested on 4.5.4 on my desktop with the old 13 profile and without forcing c++11 (obviously c++17 wasn't supported) It didn't worked (maybe It was me), on the 17.0 profile It works even without it (maybe because GCC7 defaults to c++11?).

Reply:
The first version of GCC supporting C++11 is GCC 4.8.5. Gentoo has masked all GCC <= 5.0, so the flag is not needed in the ebuild anymore. I tested with GCC 5, 6, and 7, and it worked. Could you please drop the flag from the ebuild? After that we can merge it. Thanks!

Reply:
BTW, why are you using such an old GCC as system compiler?

Reply:
Now I've removed the flag, I rarely used my old desktop and I "revived" it for this experiment I think I'm going to update it soon or later but It's  an old celeron and it's so slow..